supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -101,13 +101,17 @@ Workaround: Rerun the deployment command, or use a local container image registr
 [discrete]
 ==== Technology Preview
 
-Use present tense in the following format:
+Use present tense in the following format, where `<TechPreviewName>` is the name of your product, component or feature:
```

Comment:
I'd suggest to add something in this intro statement to make it clearer that writers are to use this text EXACTLY as is (just inserting the name of the feature at the beginning).

Reply:
Done.